src/test/java/spoon/test/main/MainTest.java

```diff
@@ -415,7 +415,6 @@ public void testTest() throws Exception {
 				//Meta model classes doesn't have to follow test class naming conventions
 				continue;
 			}
-			assertTrue(t.getQualifiedName() + " is not clearly a test class, it should contain 'test' either in its package name or class name", t.getQualifiedName().matches("(?i:.*test.*)"));
```

Comment:
this assertion is important :-)

we must move the classes into a *test* package.